R/db-postgres.r

```diff
@@ -11,27 +11,30 @@ db_desc.PostgreSQLConnection <- function(x) {
 sql_translate_env.PostgreSQLConnection <- function(con) {
   sql_variant(
     sql_translator(.parent = base_scalar,
-      log = function(x, base = exp(1)) {
-        if (isTRUE(all.equal(base, exp(1)))) {
-          build_sql("ln(", x, ")")
-        } else {
-          # Use log change-of-base because postgres doesn't support the
-          # two-argument "log(base, x)" for floating point x.
-          build_sql("log(", x, ") / log(", base, ")")
-        }
-      }
+      sd     = sql_prefix("stddev_samp"),
+      log    = function(x, base = exp(1)) {
+                  if (isTRUE(all.equal(base, exp(1)))) {
```

Comment:
So sorry, I'm not able to discern what I need to revert.  Should I put the indentation in the translation back how it was before this PR?